packages/components/radio/src/radio-button.vue

```diff
@@ -49,12 +49,10 @@ const ns = useNamespace('radio')
 const { radioRef, focus, size, disabled, modelValue, radioGroup, actualValue } =
   useRadio(props)
 
-const activeStyle = computed<CSSProperties>(() => {
-  return {
-    backgroundColor: radioGroup?.fill || '',
-    borderColor: radioGroup?.fill || '',
-    boxShadow: radioGroup?.fill ? `-1px 0 0 0 ${radioGroup.fill}` : '',
-    color: radioGroup?.textColor || '',
-  }
-})
+const activeStyle = computed<CSSProperties>(() => ({
+  backgroundColor: radioGroup?.fill ?? '',
+  borderColor: radioGroup?.fill ?? '',
+  boxShadow: radioGroup?.fill ? `-1px 0 0 0 ${radioGroup.fill}` : '',
+  color: radioGroup?.textColor ?? '',
+}))
```

Comment:
I don't think it's necessary to change this. It's clearer to not omit the braces for complex results.